code/game/objects/items/devices/mines.dm

```diff
@@ -526,6 +615,76 @@
 	desc = "An anti-infantry device produced during the corporate wars. The explosive payload has been swapped out for 'viscerator'-type antipersonnel drones."
 	spawn_type = /mob/living/simple_animal/hostile/viscerator
 
+
+
+///Claymores
+///shrapnel based dir explosive, extreme short range
+///FRONT TOWARDS ENEMY
+/obj/item/mine/directional/claymore
+	name = "C-10 Claymore"
+	desc = "A compact anti-personnel device with a directional trigger that responds to movement. A faded sticker on the back reads \"FRONT TOWARDS ENEMY\"."
+	icon = 'icons/obj/world/landmine.dmi'
```

Comment:
do not do world icons for either of these, they should be visible.

Reply:
they are- I stuck the "planted" state from the original claymore sprite source in as the world icon

Reply:
normal landmine world icons aren't implemented rn